modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/bookmark/BookmarkHelper.kt

```diff
@@ -67,6 +70,12 @@ class BookmarkHelper @Inject constructor(
             }
         }
     }
+
+    private fun shouldAllowAddBookmark(): Boolean {
+        return settings.getCachedSubscription()?.let { subscriptionStatus ->
+            subscriptionStatus is SubscriptionStatus.Paid && subscriptionStatus.tier == SubscriptionTier.PATRON
```

Comment:
Oh, it was just to type cast so that I could access the tier. I changed it slightly in https://github.com/Automattic/pocket-casts-android/pull/1274/commits/84701c44636a882781021e37df110845b6a57099 so that it is more clear.